src/platform/datapath_winuser.c

```diff
@@ -76,13 +76,13 @@ CxPlatFuzzerRecvMsg(
 //
 #define URO_MAX_DATAGRAMS_PER_INDICATION    64
 
-static_assert(
+CXPLAT_STATIC_ASSERT(
     sizeof(QUIC_BUFFER) == sizeof(WSABUF),
     "WSABUF is assumed to be interchangeable for QUIC_BUFFER");
-static_assert(
+CXPLAT_STATIC_ASSERT(
     FIELD_OFFSET(QUIC_BUFFER, Length) == FIELD_OFFSET(WSABUF, len),
     "WSABUF is assumed to be interchangeable for QUIC_BUFFER");
-static_assert(
+CXPLAT_STATIC_ASSERT(
```

Comment:
Same clang-cl error here. Curiously the 1st `CXPLAT_STATIC_ASSERT()` on line 79 is no sweat for clang-cl.